packages/flutter/lib/src/rendering/editable.dart

```diff
@@ -466,8 +457,8 @@ class RenderEditable extends RenderBox with RelayoutWhenSystemFontsChangeMixin,
   }
 
   // Caret Painters:
-  // The floating painter. This painter paints the regular caret as well.
-  late final _FloatingCursorPainter _caretPainter = _FloatingCursorPainter(_onCaretChanged);
+  // A single painter for both the regular caret and the floating cursor.
+  late final _CaretPainter _caretPainter = _CaretPainter();
```

Comment:
We've been using the words "caret" and "cursor" interchangeably.  Should we just use only "cursor" and drop "caret"?  Or how are you distinguishing the two?

Reply:
Ditto (I kept "cursor" for the floating cursor).

But indeed this is a bit messy - searched in Flutter tree:
editable* files:
- caret: 418 results in 7 files
- cursor: 1539 results in 8 files

text* files
- caret: 555 results in 20 files
- cursor: 894 results in 28 files

So no clear preference pattern :).